### PR TITLE
Fragments first section

### PR DESCRIPTION
**Pull request by tmeasday**

### CODE REVIEW

File: source/fragments.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  people(id: "7") {
+    ...NameParts
+    avatar(size: LARGE)
+  }
+}
+```
+
+There are two principal uses for fragments in Apollo:
+
+  - Sharing logic between multiple queries, mutations or subscriptions.
```
Comment (stubailo) [inline source/fragments.md]: Logic is probably the wrong word here.

File: source/fragments.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  - Breaking your queries up to allow you to co-locate field access with the places they are used.
+
+In this document we'll patterns to do both; we'll also make use of a helper package [`graphql-fragments`](https://github.com/apollostack/graphql-fragments) which aims to help us, especially with the second problem.
```
Comment (stubailo) [inline source/fragments.md]: "we'll patterns"

File: source/fragments.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+<h2 id="reusing-fragments">Reusing Fragments</h2>
+
+The most straightforward use of Fragments is to reuse parts of queries (or mutations or subscriptions) in various parts of your application. For instance, in GitHunt on the comments page, we want to fetch the same fields after posting a comment as we originally query. This way we can be sure that we render consistent comment objects as the data changes.
```
Comment (stubailo) [inline source/fragments.md]: "Fragments" shouldn't be capitalized here

File: source/fragments.md (hunk 1 above)
Comment (tmeasday) [inline source/fragments.md]: fields I guess?

Comment (stubailo) [inline source/fragments.md]: Yeah I think so

**Comment by stubailo** [comment]
🎉

**Pull request merged**